test/e2e/query_frontend_test.go

```diff
@@ -409,12 +434,13 @@ func TestQueryFrontendMemcachedCache(t *testing.T) {
 	testutil.Ok(t, err)
 	t.Cleanup(e2ethanos.CleanScenario(t, e))
 
-	now := time.Now()
+	// Predefined timestamp
+	now := time.Date(2023, time.December, 22, 12, 0, 0, 0, time.UTC)
```

Comment:
calling this now is a bit weird, but thats not super important

Reply:
right, it's kind of misleading, also is there a standard predefined timestamp that's followed in Thanos, or the one specified is fine?

Reply:
it was misleading, so changed

Reply:
awesome, thank you!